### ISSUE TITLE
Can't Join call from Side Bar

### ISSUE THREAD

**Issue by CyborgMaster** [speak/issues#191]
It used to be that if two people were talking, I would see that in the side bar and could join their call by clicking on the group.  Now I have to open up the main window (which I don't normally use) in order to join.  Please bring this functionality back to the sidebar :smile:

**Comment by mscottford** [comment]
And a related point (which might qualify as a different issue), there's no visual indication on the sidebar that those people are chatting.

**Comment by CyborgMaster** [comment]
Agreed.  That used to work and was highly valuable.  I would say one of the most important features of the sidebar.  The regression is disappointing.
